pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -320,6 +321,14 @@ type HTTPProtocol struct {
 	Path string `json:"path,omitempty" protobuf:"bytes,3,opt,name=path"`
 }
 
+// TLSProtocol matches TLS requests with specific SNI. If the field is not provided, this
```

Comment:
I follow the style of HTTP in the following:

```
// HTTPProtocol matches HTTP requests with specific host, method, and path. All
// fields could be used alone or together. If all fields are not provided, this
// matches all HTTP requests.
type HTTPProtocol struct {
	// Host represents the hostname present in the URI or the HTTP Host header to match.
	// It does not contain the port associated with the host.
	Host string
	// Method represents the HTTP method to match.
	// It could be GET, POST, PUT, HEAD, DELETE, TRACE, OPTIONS, CONNECT and PATCH.
	Method string
	// Path represents the URI path to match (Ex. "/index.html", "/admin").
	Path string
}
```

Is the comment "TLS requests" confused in some degree? If so, could you give some suggestions? Thanks a lot @jianjuns

Reply:
Perhaps say TLS handshake?

Reply:
I think `TLS handshake` is okay for me.

Reply:
Is it "TLS handshake requests"?

Reply:
I searched for the words `TLS handshake requests` but found no usage of `TLS handshake` and `requests` together. How about just `TLS handshake`?

Reply:
Then let us use "handshake packets". "handshake" is a process that includes rounds of message exchanges.